src/VisualStudio/CSharp/Impl/CSharpVSResources.resx

```diff
@@ -408,4 +408,16 @@
   <data name="NewLinesForBracesProperty" xml:space="preserve">
     <value>Place open brace on new line for properties, indexers, and events</value>
   </data>
+  <data name="QualifyMemberEventAccessWithThis" xml:space="preserve">
+    <value>Qualify member event access with 'this'</value>
```

Comment:
I don't think "member" is needed in any of these. "Qualify event access with 'this'" reads much better to me.